sockeye/arguments.py

```diff
@@ -821,10 +838,25 @@ def add_training_args(params):
                               help='How to normalize the loss. By default loss is normalized by the number '
                                    'of valid (non-PAD) tokens (%s).' % C.LOSS_NORM_VALID)
 
+    train_params.add_argument('--length-task',
```

Comment:
Would it make sense to rename to make clearer that its about length prediction? `--predict-length` or `length-prediction-task` ...

Reply:
Yeah, it was initially called ```--length-ratio...``` but, since it can already predict float ratios or integer lengths depending on the loss, it was a bit confusing, so was changed to just ```--length-task..```. Felix also pointed out that different length-related tasks might be implemented in the future.